### DISCUSSION TITLE
Light terminal theme support

### DISCUSSION THREAD

**Discussion by nicck** [inv2004/ttop#20] [Q&A] [answered]
I've tried to use it with light terminal theme (Alabaster) and it took me some to realize columns are not actually empty. Are colors hardcoded? Is it possible to override colors via config somehow?

![image](https://user-images.githubusercontent.com/14026/222985421-e424a17c-90d5-489d-9d62-4c68ab259dea.png)

**Comment by inv2004** [comment, marked as answer]
`ttop.toml` option `light = true` for light-mode in 0.9.1 version
https://github.com/inv2004/ttop/issues/22#issuecomment-1464556557
